### ISSUE TITLE
No responsiveness for calculator page

### ISSUE THREAD

**Issue by t-muthuri** [t-muthuri/building-plans-approvals-fe#3]
# Description
The ui for the calculator page is designed for a wide screen

# Expected Behavior
The calculator page responds to different screen sizes

# Actual Behavior
The calculator page is not responsive

# Motivation
improved ui

**Comment by Rodney1988** [comment]
hi @t-muthuri , can I check this one?

**Comment by t-muthuri** [comment]
Yes. Thank youOn 17 Aug 2024, at 00:10, Rodrigo ***@***.***> wrote:﻿
hi @t-muthuri , can I check this one?
